src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+const logo = document.querySelector('.logo');
+
+const successHandler = () => {
+  const messageSuccess = document.createElement('div');
+
+  messageSuccess.classList.add('message');
+  messageSuccess.textContent = 'Promise was resolved!';
+  body.appendChild(messageSuccess);
+};
+
+const errorHandler = () => {
+  const messageError = document.createElement('div');
+
+  messageError.classList.add('message');
+  messageError.textContent = 'Promise was resolved!';
```

Comment:
```suggestion
  messageError.classList.add('error-message');
  messageError.textContent = 'Promise was rejected!';
```
it should fix tests

Reply:
also successHandler and errorHandler have same logic and for them can be one util function